custom_components/tuya_local/devices/klarta_stor2_purifier.yaml

```diff
@@ -0,0 +1,149 @@
+name: Klarta Stor 2 air purifier
+products:
+  - id: ku6ae3y8ad6tbx3b
+    name: "KLARTA STOR 2 "
+primary_entity:
+  entity: switch
```

Comment:
This would be better represented as a fan, along with speed and preset_mode dps.